service/src/main/resources/application.yaml

```diff
@@ -154,12 +154,12 @@ quarkus:
   datasource:
     db-kind: postgresql
     jdbc:
-      max-size: 10
+      max-size: 15
```

Comment:
All these values make sense to me, there is nothing to loose by trying them and we can look to gather more detailed logging/metrics from the datasource in following PRs